docs/TOOLS.md

```diff
@@ -0,0 +1,57 @@
+## Code Style and Linting
+
+If you are looking for good general advices regarding Perl coding, 
+Damian Conway's [Perl Best Practices][PBP] is a good reference. 
```

Comment:
Shoot. I didn't check the file name. We don't reference `TOOLS.md` anymore:

https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#track-anatomy

I'll rename this to `RESOURCES.md`.